doc/colvars-refman-main.tex

```diff
@@ -550,9 +606,29 @@
 
 For the output files, the prefix of Colvars output files will use the \texttt{-defnnm} option of mdrun, or be the same as the GROMACS log file if the option is not set.  Otherwise, the prefix will be \emph{output}.
 
-\textbf{Note on PBC wrapping in GROMACS.} Internally, GROMACS handles only PBC-wrapped atom coordinates, which may break the calculation of some variables if their atom groups become split across PBCs.
-To prevent this, Colvars unwraps coordinates throughout the simulation, by assuming that each atom group is intact in the initial coordinates, and canceling any later jumps across the periodic box.
-This information is propagated across restarts, and communicated between replicas when GROMACS is used for replica-exchange simulations. Thus, Colvars is compatible with native replica-exchange in GROMACS.
+\textbf{Note:} \emph{This section describes how to use Colvars in GROMACS versions starting 2024, which support and include Colvars natively.}
+
+To enable a Colvars-based calculation, just modify the \texttt{mdp} file to add those two keywords:
+\begin{mdexampleinput}
+  \-...\\
+  \-colvars-active~=~yes\\
+  \-colvars-configfile~=~my\_colvars.dat\\
+  \-...\\
+\end{mdexampleinput}
+
+Then, create the \texttt{tpr} and launch the simulation as usual:
+\begin{mdexampleinput}
+  \-gmx~grompp~-f~system.mdp~-p~system.top~-c~init.gro~-o~test.tpr
+  \-gmx~mdrun~-s~test.tpr~-deffnm~md~
+\end{mdexampleinput}
+
+Restarting from a previous simulation is done using the \texttt{-cpi} parameter for the checkpoint file (e.g.{} ``\texttt{state.cpt}''). This file holds the required information to restart the Colvars-based simulation.
+\begin{mdexampleinput}
+  \-gmx~mdrun~-s~test\_restart.tpr~-deffnm~md~-cpi~state.cpt
+\end{mdexampleinput}
+
+For the output files, the prefix of Colvars output files share the same prefix as the \texttt{edr} file, which is ``\texttt{ener}'' by default. This is modified if the \texttt{-defnnm} or the \texttt{-e} option of mdrun is used.
```

Comment:
This section looks like it is in the wrong branch of the `\ifdefined\gromacsmdmodule` condition - or am I missing something?